doc/devdocs/ast.rst

```diff
@@ -267,7 +267,7 @@ Input                    AST
 ``a:b``                  ``(: a b)``
 ``a:b:c``                ``(: a b c)``
 ``a,b``                  ``(tuple a b)``
-``a==b``                 ``(comparison a == b)``
+``a==b``                 ``(call == a b)``
```

Comment:
`git rebase -i origin/master` and deselect spencer's commit, then force push over this `vs/curl` branch